lib/puppet/type/cisco_itd_service.rb

```diff
@@ -0,0 +1,338 @@
+# Manages a Cisco ItdService.
+#
+# March 2016
+#
+# Copyright (c) 2015-2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+Puppet::Type.newtype(:cisco_itd_service) do
+  @doc = "Manages a Cisco ItdService.
+
+  Any resource dependency should be run before the interface resource.
+
+  cisco_itd_service {\"<name>\":
+    ..attributes..
+  }
+
+  <name> is the complete name of the service.
+
+  Example:
+    cisco_itd_service {\"my_service\":
+     ensure                        => present,
+     access_list                   => \"my_access1\",
```

Comment:
You don't need the \" for there.  Change these to use the following format:  `access_list   => 'my_access`',